### PR TITLE
Sync latest changes of pluginv2 proto

### PR DESCRIPTION
**Pull request by marefr**
/api/plugins/grafana-image-renderer/health now works again.

### CODE REVIEW

File: proto/pluginv2.proto [hunk 1]
```diff
@@ -7,44 +7,59 @@ option go_package = ".;pluginv2";
...
-
 message User {
   string login = 1;
   string name = 2;
   string email = 3;
   string role = 4;
 }
 
+message PluginContext {
+  // The Grafana organization id the request originating from.
```
Comment (aknuds1) [inline proto/pluginv2.proto:37]:
```suggestion
  // The Grafana organization id the request originates from.
```

File: proto/pluginv2.proto [hunk 2]
```diff
@@ -7,44 +7,59 @@ option go_package = ".;pluginv2";
...
+  int64 orgId = 1;
+
+  // The unique identifier of the plugin the request  originating from.
```
Comment (aknuds1) [inline proto/pluginv2.proto:40]:
```suggestion
  // The unique identifier of the plugin the request originates from.
```

File: proto/pluginv2.proto [hunk 3]
```diff
@@ -7,44 +7,59 @@ option go_package = ".;pluginv2";
...
+  string pluginId = 2;
+
+  // The Grafana user the request originating from.
```
Comment (aknuds1) [inline proto/pluginv2.proto:43]:
```suggestion
  // The Grafana user the request originates from.
```

File: proto/pluginv2.proto (hunk 1 above)
Comment (marefr) [reply proto/pluginv2.proto:37]: This should be changed in https://github.com/grafana/grafana-plugin-sdk-go/blob/master/proto/backend.proto

File: proto/pluginv2.proto [hunk 4]
```diff
@@ -7,44 +7,59 @@ option go_package = ".;pluginv2";
...
+  //
+  // Will not be provided if Grafana backend initiated the request.
+  User user = 3;
+
+  // App plugin instance settings is the configured app instance settings.
+  // In Grafana an app instance is an enabled app plugin in a
+  // Grafana organization.
+  //
+  // Will only be set if request targeting an app instance.
```
Comment (aknuds1) [inline proto/pluginv2.proto:52]:
```suggestion
  // Will only be set if request is targeting an app instance.
```

File: proto/pluginv2.proto [hunk 5]
```diff
@@ -7,44 +7,59 @@ option go_package = ".;pluginv2";
...
+  AppInstanceSettings appInstanceSettings = 4;
+
+  // Data source instance settings is the configured data source instance
+  // settings. In Grafana a data source instance is a created data source
+  // in a Grafana organization.
+  //
+  // Will only be set if request targeting a data source instance.
```
Comment (aknuds1) [inline proto/pluginv2.proto:59]:
```suggestion
  // Will only be set if request is targeting a data source instance.
```

File: proto/pluginv2.proto (hunk 1 above)
Comment (aknuds1) [reply proto/pluginv2.proto:37]: @marefr OK thanks for the heads up.

**Comment by marefr** [comment]
Yes, this is from sdk master

**Pull request merged**